content/trial_getting_started/examples/automation_tools/aws_terraform_basics.md

```diff
@@ -66,7 +66,7 @@ To store the access keys as secrets in the Cloudify manager, login to the {{< pa
 
 ### Upload Plugins
 
-Plugins are Cloudify's extendable interfaces to services, cloud providers and automation tools.
+Plugins are {{< param product_name >}}'s extendable interfaces to services, cloud providers and automation tools.
 I.e., connecting to {{< param cloud >}} requires the {{< param cloud >}} plugin.
 
 To upload the required plugins to your manager, select the **Cloudify Catalog** page, scroll to the **Plugins Catalog** widget and select the plugins you wish to upload.
```

Comment:
Here (as well as on others pages in examples series) we have "Cloudify Catalog" page. Do we want to do anything with it?

Reply:
No. I kept this one on purpose because changing it will not modify the UI part and they re interleaved.